Feature/branch fessko

fessko opened the pull request:
Обновил название ветки, понизил версию файлов до версии R2021b

fessko committed:
Create train_model_fessko1.slx

Создание модели колебаний математического маятника

fessko committed:
Create train_model_fessko2.slx

Сделал модель математического маятника с затуханием, вроде как с выполнением всех пунктов поставленной задачи

fessko committed:
Понижение до версии R2021b

fessko committed:
Внесение мелких изменений

Изменена компановка, добавлено название линии

NoMonsters commented:
### Модель без затухания
Всё окей, только крайне желательно не поворачивать блоки назад. Смешанный тип в модели, когда ещё надо читать и слева направо и справа налево затруднителен. В простых моделях неважно, но в крупных усложняет восприятие.

### Модель с затуханием
Окей, работает корректно.
- Почему решил использовать схему с вектором производных а не скалярный вариант схемы?
-  Нужно задать тип самих входов подсистемы
- Не нашел запись в SDI, есть только out в workspace тоже прикольно, но не совсем то. Запись в SDI можно включить нажав на сигнал и выбрав опцию log selected signal

Пару этих моменто исправь, пожалуйста и будем приступать к ДУ самолета

fessko committed:
Внесение изменений

Задан тип данных входов подсистемы, выполнена запись основных параметров в SDI, удален out в workspace

fessko commented:
Решил использовать схему с вектором производных из-за того, что подобное решение ДУ было рассмотрено на практике по Simulink в ВУЗе

fessko committed:
Update train_model_fessko_2.slx

NoMonsters reviewed (approved):
окей объединяем
